bittensor/_neuron/text/sgmoe_validator/run.py

```diff
@@ -156,6 +161,7 @@ def run( config , validator, subtensor, wallet, metagraph, dataset, device, uid,
             ], axis = 1)
             df['uid'] = df.index
             wandb_dendrite = dendrite.to_wandb()
+            #wandb.log(info, step =current_block)
```

Comment:
is this intentional? if so, let's just take out the whole comment.

Reply:
Oops, let me remove that